tests/test_restore_names.py

```diff
@@ -1,3 +1,21 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_function_should_return_none():
+    assert restore_names([]) is None
```

Comment:
If I right understand our function should return None anyway. Correct me, if it's not (https://prnt.sc/26omq2e)

Reply:
I think you did not fully understand the condition. My check is that the function does not return anything. And if it doesn't return anything specific, it returns None by default. This is the point of this test. So if function will return something, there will be an error.

Reply:
> 

Ok. Understand. Thank you